loguru_mypy/__init__.py

```diff
@@ -33,19 +36,45 @@
     'Named argument of loguru handler is not valid for given message',
     'loguru',
 )
+ERROR_BAD_RECORD: te.Final[ErrorCode] = ErrorCode(
+    'logger-record',
+    'Invalid access to loguru record structure',
+    'loguru',
+)
 
 
 class Opts(t.NamedTuple):
     lazy: bool
+    record: bool
 
 
 DEFAULT_LAZY = False  # type: te.Final
-DEFAULT_OPTS = Opts(lazy=DEFAULT_LAZY)  # type: te.Final
+DEFAULT_RECORD = False  # type: te.Final
+DEFAULT_OPTS = Opts(
+    lazy=DEFAULT_LAZY,
+    record=DEFAULT_RECORD,
+)  # type: te.Final
 NAME_TO_BOOL = {
     'False': False,
     'True': True,
 }  # type: te.Final
 
+RECORD_ARGS = (
```

Comment:
BTW: @Delgan it is pity that you cannot exactly import `.pyi` files. It would make more sense to have it done like that to import from currently available `loguru`.